python/ray/dataframe/dataframe.py

```diff
@@ -21,29 +23,51 @@ def __init__(self, df, columns):
         assert(len(df) > 0)
 
         self._df = df
+        self._lengths = \
+            ray.get([_deploy_func.remote(_get_lengths, d) for d in self._df])
```

Comment:
This is going to be cleaned up later. We will call `get` only when we access the object (and only once).